cloudapp.gemspec

```diff
@@ -43,17 +43,20 @@ Gem::Specification.new do |s|
 
   ## List your runtime dependencies here. Runtime dependencies are those
   ## that are needed for an end user to actually USE your code.
-  s.add_dependency 'clipboard',  '~> 1.0.1'
-  s.add_dependency 'leadlight',  '~> 0.1.0'
-  s.add_dependency 'mime-types', '~> 1.19'
-  s.add_dependency 'netrc',      '~> 0.7.7'
-  s.add_dependency 'typhoeus',   '~> 0.7.3'
+  s.add_dependency "clipboard", "~> 1.0.1"
+  s.add_dependency "leadlight", "~> 0.1.0"
+  s.add_dependency "mime-types", "~> 1.19"
+  s.add_dependency "netrc", "~> 0.7.7"
+  s.add_dependency "typhoeus", "~> 0.7.3"
+  s.add_dependency "http", "~> 0.7.4"
+  s.add_dependency "terminal-table"
+  s.add_dependency "pry"
```

Comment:
@macsrok `pry` should not be a runtime dependency, can you move it to development deps ?